config/startup.options

```diff
@@ -0,0 +1,57 @@
+################################################################################
+# These settings are ONLY used by $LS_HOME/bin/system-install to create a custom
```

Comment:
If this file is only used by system-install, maybe we can combine the logstash jvm options w/ a new file maybe called 'service.options' or something, and that file probably only has two settings (the name and description of the service?)